<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFExportToTableDialog</class>
 <widget class="QDialog" name="QFExportToTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add plot data to table ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,0,0,1,0">
   <item>
    <widget class="QTextBrowser" name="labDescription">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>column name prefix:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edtPrefix">
       <property name="toolTip">
        <string>this text is prepended to every column name in the table (e.g. use this to identify where which column came from!)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtNewTableName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkNewTable">
       <property name="text">
        <string>create new:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFRDRTableComboBox" name="cmbRDR"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>file:</string>
       </property>
       <property name="buddy">
        <cstring>cmbRDR</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkShowTableEditor">
     <property name="text">
      <string>show table editor after insert</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFRDRTableComboBox</class>
   <extends>QComboBox</extends>
   <header>qfrdrtablecombobox.h</header>
   <slots>
    <signal>currentColumnGraphicsChanged(QFRDRColumnGraphsInterface*)</signal>
    <signal>currentTableChanged(QFRDRTableInterface*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFExportToTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFExportToTableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
